mir_dataset_loaders/beatles.py

```diff
@@ -0,0 +1,203 @@
+"""Beatles Dataset Loader
+"""
+from collections import namedtuple
+import json
+import numpy as np
+import os
+import csv
+
+from . import BEATLES_INDEX_PATH
+from .utils import (get_save_path, get_local_path, validator, BeatData, ChordData, SectionData, KeyData,
+                    download_from_remote, RemoteFileMetadata, untar)
+
+BEATLES_DIR = 'Beatles'
+BEATLES_INDEX = json.load(open(BEATLES_INDEX_PATH, 'r'))
+BEATLES_ANNOT_REMOTE = RemoteFileMetadata(  # TODO: @rabbit this is not metadata but the annotations, do especific data type?
```

Comment:
I think the name `BEATLES_ANNOT_REMOTE` is totally ok, if this is what you're asking :)

Reply:
Just asking if you want to be extra consistent with data types and want me to create a `RemoteFileAnnotations` (which will be the same as `RemoteFileMetadata` with another name)